index.js

```diff
@@ -57,11 +66,26 @@ module.exports = {
         depth: 3,
       },
     ],
+    'simple-import-sort/sort': 'error',
```

Comment:
@lofgrenfredrik Could you explain a little bit what these rules do?

Reply:
@michaellopez Added a separate comment including short list and explanation of the activated rules. 👍